staff/Melinx/css-adaptive/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/main.css">
+   
+</head>
+
+<body>
+
+    <header>header</header>
+    <main>
+        <nav class="col">nav</nav>
```

Comment:
WHAT do you need selector ```col```for? AFAIS not used in styles...

Reply:
true that. Removed class "col"